apps/astarte_pairing_api/lib/astarte_pairing_api_web/router.ex

```diff
@@ -20,17 +20,17 @@ defmodule Astarte.Pairing.APIWeb.Router do
   use Astarte.Pairing.APIWeb, :router
 
   pipeline :api do
-    plug :accepts, ["json"]
```

Comment:
The `:accepts, ["json"]` plug should be included in this pipeline too, as e.g. the `"/agent"` scope needs it

Reply:
The `accepts, [json]` is already added in `api_device` pipeline [accepts](https://github.com/osmanhadzic/astarte/blob/736eb8bdf77b8f92bbd183d9e49d5507c5408457/apps/astarte_pairing_api/lib/astarte_pairing_api_web/router.ex#L28)

Reply:
Ops, didn't notice, thank you